app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less

```diff
@@ -1,246 +1,252 @@
-        font-weight: 300;
-        text-transform: uppercase;
-        &::before {
-            display: none;
+            .lib-font-size(12);
+            font-weight: @font-weight__light;
+            text-transform: uppercase;
+            &::before {
+                display: none;
+            }
         }
-    }
-    .slick-next {
-        position: absolute;
-        top: 0;
-        right: 0;
-        &::before {
-            display: flex;
-            align-items: center;
-            content: "|";
+        .slick-next {
+            position: absolute;
+            top: 0;
+            right: 0;
+            &::before {
+                .lib-vendor-prefix-display(flex);
+                align-items: center;
+                content: "|";
+            }
+        }
+        .slick-prev {
+            position: absolute;
+            top: 0;
+            right: 0;
+            overflow: hidden;
         }
     }
-    .slick-prev {
-        position: absolute;
-        top: 0;
-        right: 0;
-        overflow: hidden;
-    }
-}
 
-/* SECTION__SALE*/
-.section__sale {
-    p {
-        color: @color-maksym__main-black;
-    }
-    .sale__banner {
-        .pagebuilder-banner-wrapper
-            .pagebuilder-overlay:not(.pagebuilder-poster-overlay) {
-            width: 100%;
+    /* SECTION__SALE*/
+    .section__sale {
+        p {
+            color: @color-maksym__main-black;
         }
-        [data-element="wrapper"] {
-            display: block;
-            width: 100%;
-            margin: 0%;
-
+        .sale__banner {
             .pagebuilder-banner-wrapper
                 .pagebuilder-overlay:not(.pagebuilder-poster-overlay) {
                 width: 100%;
             }
-        }
-        .pagebuilder-overlay {
-            display: block;
-            width: 100%;
-            height: 100%;
-            margin: 0;
-            &:not(.pagebuilder-poster-overlay) {
+            [data-element="wrapper"] {
+                display: block;
                 width: 100%;
+                margin: 0%;
+
+                .pagebuilder-banner-wrapper
+                    .pagebuilder-overlay:not(.pagebuilder-poster-overlay) {
+                    width: 100%;
+                }
             }
-            .pagebuilder-collage-content {
+            .pagebuilder-overlay {
+                display: block;
                 width: 100%;
-                display: flex;
-                justify-content: space-between;
+                height: 100%;
+                margin: 0;
+                &:not(.pagebuilder-poster-overlay) {
+                    width: 100%;
+                }
+                .pagebuilder-collage-content {
+                    width: 100%;
+                    .lib-vendor-prefix-display(flex);
+                    justify-content: space-between;
+                }
+            }
+            [data-element="content"] {
+                .lib-vendor-prefix-display(flex);
+                flex-direction: column;
```

Comment:
there is a lib for flex-direction:
`.lib-vendor-prefix-flex-direction();`

Reply:
done